pytransform3d/transform_manager/_temporal_transform_manager.py

```diff
@@ -75,17 +128,14 @@ def _transform_available(self, key):
         return key in self._transforms
 
     def _set_transform(self, key, A2B):
-        if not isinstance(A2B, TimeTransform):
-            A2B = StaticTransform(A2B)
         self._transforms[key] = A2B
 
     def _get_transform(self, key):
-        return self.transforms[key]
```

Comment:
I would like to avoid constructing the whole list of transforms on every call to `get_transform`.

Reply:
Agree